app/views/home/_navbar.html.erb

```diff
@@ -0,0 +1,25 @@
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <div class="container-fluid">
+    <%= link_to "Recipe App", public_recipes_path, class:"navbar-brand"%>
```

Comment:
- [ ] [OPTIONAL] I guess you meant to name this `public recipe` not `recipe app` kindly affect the change since it is a link to a public recipe page so as to avoid confusion :+1: